setup.py

```diff
@@ -6,7 +6,7 @@
 
 setup(
     name="xem_wrapper",
-    version="0.2.5",
+    version="0.3.0",
     description="Functions for interacting with a XEM device",
     url="https://github.com/CuriBio/xem-wrapper",
     author="Curi Bio",
```

Comment:
this was the part of the metadata I was referencing to drop
![image](https://user-images.githubusercontent.com/5436635/110864629-6406c980-8290-11eb-90a8-8ed1fbf765be.png)